docs/gravitino-server-config.md

```diff
@@ -63,6 +63,17 @@ You can also specify filter parameters by setting configuration entries of the f
 We strongly recommend that you change the default value of `gravitino.entity.store.kv.rocksdbPath`, as it's under the deployment directory and future version upgrades may remove it.
 :::
 
+### Tree lock configuration
+
+Gravitino server uses tree lock to ensure the consistency of the data. The tree lock is a memory lock (Currently, Gravitino only supports in memory lock) that can be used to ensure the consistency of the data in Gravitino server. for more impormation about tree lock, please refer to design [doc1](https://docs.google.com/document/d/1fAxwWdgOHNouysJNgilI5WbCzF1MiTVqsaiGiUga5VA/edit#heading=h.f8rd8hp69vwq) and [doc2](https://docs.google.com/document/d/1ov81tnkRAUkac-K8AjfTsK2AAbpCfnSBfE9XqrTlyWc/edit).
```

Comment:
Remove the design doc here, don't put the design doc into the user doc, it is unrelated.

Reply:
done